.php_cs.dist

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$config = new PrestaShop\CodingStandards\CsFixer\Config();
```

Comment:
```suggestion
$config = new PhpCsFixer\Config();
```

Reply:
Nope, we need to include "our" Cs Fixer configuration.

Reply:
Then you have to fix the require for PrettyCI. :)